project.clj

```diff
@@ -10,16 +10,21 @@
 
   :description "SSL certificate management on the JVM."
 
+  :min-lein-version "2.7.1"
+
+  :parent-project {:coords [puppetlabs/clj-parent "0.2.5"]
+                   :inherit [:managed-dependencies]}
+
   ;; Abort when version ranges or version conflicts are detected in
   ;; dependencies. Also supports :warn to simply emit warnings.
-  ;; requires lein 2.2.0+.
   :pedantic? :abort
 
-  :dependencies [[org.clojure/tools.logging "0.2.6" :exclusions [org.clojure/clojure]]
+  :dependencies [[org.clojure/tools.logging]
                  [org.bouncycastle/bcpkix-jdk15on "1.55"]
-                 [commons-codec "1.9"]
-                 [clj-time "0.7.0"]
```

Comment:
If `ssl-utils` had a different version dependency range from what it is inheriting from the specific version of `clj-parent` it is pulling in, then I think it would make sense to have an explicit dependency here.  That doesn't seem to be the case here, though.  If we did continue to have an explicit dependency for it here, then we could have all of the same problems with needing resolve conflicting transitive dependencies that `clj-parent` was intended to help resolve for downstream consumers.  Removing the version here seems consistent with what we've done elsewhere in updating components to the use of `clj-parent`.